trin-state/src/jsonrpc.rs

```diff
@@ -18,6 +23,16 @@ impl StateRequestHandler {
                     let radius = &self.network.overlay.data_radius;
                     let _ = request.resp.send(Ok(Value::String(radius.to_string())));
                 }
+                StateEndpoint::Ping => {
+                    let response = match PingParams::try_from(request.params) {
+                        Ok(val) => match self.network.overlay.send_ping(val.enr, None).await {
+                            Ok(pong) => Ok(Value::String(format!("{:?}", pong.payload.unwrap()))),
```

Comment:
Looks like an invalid pong payload would crash trin here, so we need a little more error handling.